python/delete_instuck_progress.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+'''
+deletes file in case it exists
+'''
+
+import os
+import os.path
+from datetime import datetime
+from datetime import date
+
+
+def drop_file(abs_folder, filename):
+    if os.path.isfile(os.path.join(abs_folder, filename)):
+        os.remove(os.path.join(abs_folder, filename))
+        return True
+    return False
+
+
+def drop_outdated_inprogress_file(abs_folder, filename):
+    if os.path.isfile(os.path.join(abs_folder, filename)):
+        modification_time = os.stat(os.path.join(abs_folder, filename)).st_mtime
+        today = date.today()
+        modification_day = date.fromtimestamp(modification_time)
+        if today > modification_day:
+            os.remove(os.path.join(abs_folder, filename))
```

Comment:
You can save this `os.path.join(abs_folder, filename)` in a variable (`full_file_path`?) and avoid doing it three times. Also it would lower the probability of introducing a bug in the future, like in case of changing the if `join` and then forgetting to change others.